README.md

```diff
@@ -1,6 +1,21 @@
 # autocomplete-nn
-week-4 project
 
+** Team members **  
+ - @Noorb7188  :sunglasses:
+ - @NaremanPro :sunglasses:
 
-team mates : @Noorb7188
-            @NaremanPro
+Before we start coding we draw our architecture to make a plan for writing our code step by step .   
+
+![]()  
+
+:scroll: First we choose a list of words called *Archaic*  words- the words that are ancient and are no longer in use . we took it from [here](https://en.oxforddictionaries.com/explore/archaic-words).  
+
+:bookmark_tabs: Then we start creating the files :
+- public : That includes the `index.html` [with all the html code and links to the other files in the public], `style.css` [styling and media query], `dom.js` [js that manipulate the dom]
+- src : That includes `handler.js`[functions we used in the router], `router.js`[that responds by endpoints], `server.js`[our listener], and the `word.txt`[list of the words].  
+
+:bookmark_tabs: Secondly we coded our `server` that listens to our `router`, then we write the functions in the router that takes the request and response, to see our public code on the port with the suitable endpoints .  
+
+:muscle: It was very challenging because we have been exposed to these materials just for day and a half, we couldn't put all the pieces together, but we tried our best to learn through coding this app.
+
+If we had more time we surely would would finish the app untill the end till the last detail , and try new things we haven't learn even like `xmlhttprequest`. :no_good:
```

Comment:
great job lady, 
you have a typing mistake in line 21 you wrote "would" twice.. and you missed the part that talks about the TDD and tape, instaling npm and running tests. 
you have done alot today, we will fix it tommorw morning and add the architecture picture too.